Network/public-ip.5s.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# <xbar.title>Public IP Status</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Hein Soe</xbar.author>
+# <xbar.author.github>hheinsoee</xbar.author.github>
+# <xbar.desc>Shows your public IP address, ISP, and location in the menu bar</xbar.desc>
+# <xbar.dependencies>bash,curl</xbar.dependencies>
+# <xbar.image>https://www.freeiconspng.com/uploads/data-network-icon-image-gallery-5.png</xbar.image>
```

Comment:
The image should be a screenshot of your plugin in action, typically what it displays with any sub-menus open.